draft-ietf-taps-interface.md

```diff
@@ -1032,15 +790,15 @@ Connection -> SendError<msgRef>
 A SendError occurs when a Message could not be sent due to an error condition:
 an attempt to send a Message which is too large for the system and
 Protocol Stack to handle, some failure of the underlying Protocol Stack, or a
-set of send parameters not consistent with the Connection's transport
-parameters. The SendError contains an implementation-specific reference to the
+set of Message Properties not consistent with the Connection's transport
+properties. The SendError contains an implementation-specific reference to the
 Message to which it applies.
 
-## Message Context Parameters {#send-params}
+## Message Context Properties {#message-props}
```

Comment:
I'd prefer to leave this category as Parameters, since they're not all "properties"

Reply:
Renamed this part back to what it was in #200, and left "Final" up as a Message Context Parameter.
I think the distinction between "parameters" and "properties" goes along the same lines as the distinction between "Control Properties" and "Protocol Properties", and I'm looking forward to the discussion in Montreal.